expat/configure.ac

```diff
@@ -276,12 +276,46 @@ AS_IF([test "x${DOCBOOK_TO_MAN}" != x -a "x$with_docbook" != xno],
 
 AM_CONDITIONAL(WITH_DOCBOOK, [test "x${DOCBOOK_TO_MAN}" != x])
 
+AC_ARG_ENABLE(stdcxx,
+AS_HELP_STRING([--enable-stdcxx], [Specify CXX standard library]),
+    [AX_APPEND_COMPILE_FLAGS([-std=${enableval}], [AM_CXXFLAGS])],[])
+
+AC_ARG_ENABLE(fuzztargets,
+    AS_HELP_STRING([--enable-fuzztargets], [Enable fuzz targets]),[enable_fuzztargets=$enableval],[enable_fuzztargets=no])
+AM_CONDITIONAL([BUILD_FUZZTARGETS], [test "x$enable_fuzztargets" = "xyes"])
+AS_IF([test "x$enable_fuzztargets" = "xyes"], [
+    AC_MSG_CHECKING([whether $CXX accepts -fsanitize=fuzzer])
+    AC_LANG_PUSH(C++)
+    tmp_saved_flags=$[]_AC_LANG_PREFIX[]FLAGS
+    _AC_LANG_PREFIX[]FLAGS="$[]_AC_LANG_PREFIX[]FLAGS -std=c++11 -fsanitize=fuzzer"
```

Comment:
I'm not sure here. @catenacyber

Reply:
I want to test if the compiler supports the flag `-fsanitize=fuzzer` as it is needed for https://github.com/libexpat/libexpat/pull/292/files#diff-d3dd4bd153655cf8432cf0587456b6afR10

I do not know if there is a nicer way to do it...

The trick is that we need to use clang++ as linker

Reply:
Something like this might work:
```
AS_VAR_COPY([OLDFLAGS],[CXXFLAGS])
AX_APPEND_FLAG([-fsanitize=fuzzer], [CXXFLAGS])
[..]
AS_VAR_COPY([CXXFLAGS],[OLDFLAGS])])
```
It's based off ideas of the code we have in the [conftools](https://github.com/libexpat/libexpat/tree/master/expat/conftools) folder.